source/chapters/1/intro.md

```diff
@@ -132,14 +155,13 @@ moment(new Date()).format('YYYY-MM-DD')
 
 ## Atmosphere
 
-Atmosphere is a package manager which is specific to Meteor only. As such, it will help you manage project dependencies with ease.
+Atmosphere is a package manager which is specific to Meteor only, just like pip is for Python. 
```

Comment:
Why it says about Pip for Python? I would rather say something more about atmosphere, not about pip.

Reply:
Changed the comparation from "pip for Python" to "npm for NodeJs". I want to use an example from another ecosystem, to create an analogy, and help the reader understand better what i'm talking about.